### PR TITLE
Create option to add customer to coach

### PR DESCRIPTION
**Pull request by Peralban**
I create some route to add and remove client in one coach client list. We can get all the client of one coach.

### CODE REVIEW

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

File: backend/routes/employee.ts [hunk 1]
```diff
@@ -13,12 +13,50 @@ async function getEmployeeFromDB(employeeId: number) {
     return employee;
 }
 
+async function custumerIsInDB(constId: number) {
```
Comment (Thomaltarix) [inline backend/routes/employee.ts:16]: custumer ?

File: backend/routes/employee.ts [hunk 2]
```diff
@@ -13,12 +13,50 @@ async function getEmployeeFromDB(employeeId: number) {
...
+        where: { id: constId }
+    });
+    return customer !== null;
+}
+
+async function getIdFromRequest(req: any) {
+    return 1;
+}
+
+async function getEmployeeConstumerList(employee: any, res: any) {
```
Comment (Thomaltarix) [inline backend/routes/employee.ts:27]: Constumer ?

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Some "little" changes (except it's maybe in all your previous files) but overall great

**Review by Thomaltarix — APPROVED** [review]
:+1:

**Pull request merged**